calculator.js

```diff
@@ -1,19 +1,67 @@
-
-const exampleAdditionInput = {
-  num1: 3,
-  num2: 5,
-  operation: 'add',
+const userInput = {
+  num1: 30,
+  num2: 10,
+  operation: 'subtract',
 }
 
+const addNums = function(a, b) {
```

Comment:
Great work encapsulating all the operation functionality into methods. Consider using an arrow functions. Arrow functions are particularly nice when the function block is only one line.